src/containers/Results/Result/index.js

```diff
@@ -147,8 +147,12 @@ const Result = ({ result, addExperiment, removeExperiment, dataSet }) => {
           <i className="result__not-provided">No associated publication</i>
         )}
       </p>
-
-      <SampleMetadataFields samples={result.samples} />
+      <h3>Sample Metadata Fields</h3>
+      <p className="result__paragraph">
+        {(metadataFields && metadataFields.join(', ')) || (
```

Comment:
Can `getMetadataFields` return `[]`?? maybe it would be good to check `metadataFields.length` as well, or ensure it doesn't return `[]`.